app/javascript/mastodon/features/audio/index.js

```diff
@@ -555,6 +556,7 @@ class Audio extends React.PureComponent {
             </div>
 
             <div className='video-player__buttons right'>
+              {!alt && <button type='button' title={intl.formatMessage(messages.no_descriptive_text)} aria-label={intl.formatMessage(messages.no_descriptive_text)} className='player-button no-action media__no-description-icon' ><Icon id='exclamation-triangle' fixedWidth /></button>}
```

Comment:
Using a `button` for something that has no action sounds slightly off to me, but I imagine that using anything else might cause subtle alignment issues with neighboring elements which *are* buttons.